pkg/handler/employee/employee.go

```diff
@@ -28,8 +29,41 @@ func New(store *store.Store, service *service.Service, logger logger.Logger) IHa
 	}
 }
 
+// List godoc
+// @Summary Get the list of employees
+// @Description Get the list of employees with pagination and working_status
+// @Tags Employee
+// @Accept  json
+// @Produce  json
+// @Param Authorization header string true "jwt token"
+// @Success 200 {object} view.EmployeeListDataResponse
+// @Failure 400 {object} view.ErrorResponse
+// @Failure 404 {object} view.ErrorResponse
+// @Failure 500 {object} view.ErrorResponse
+// @Router /employees [get]
```

Comment:
I think we need a test for this function

Reply:
oke